mkdocs.yml

```diff
@@ -0,0 +1,68 @@
+site_name: fiware-cygnus
+site_url: https://fiware-cygnus.readthedocs.org
+repo_url: https://github.com/telefonicaid/fiware-cygnus.git
+site_description: Cygnus Documentation
+docs_dir: doc
+site_dir: html
+markdown_extensions: [toc,fenced_code]
+use_directory_urls: false
+theme: readthedocs
+extra_css: ["https://fiware.org/style/fiware_readthedocs.css"]
+pages:
+  - Home: index.md
+  - 'Contributing guidelines': 'contributing/contributing_guidelines.md'
```

Comment:
perhaps we can link this from index.md, by having a section on index.md and a link to this document from that section, what do you think?

I find it a bit weird to have the contribution guidelines so high in the readthedocs TOC.